yabt/artifact.py

```diff
@@ -70,6 +71,10 @@ class ArtifactStore:
     - "gen_h"   A generated C/C++ header file (e.g. from Proto's) that is used
                 by any dependent C++ builder (direct or indirect) to extend
                 the target headers. Cached as ouputs.
+    - "gen_go"  A generated Go source file (e.g. from Proto's) that is used
+                only by directly dependent Go builders (used to extend
```

Comment:
given the comment about wanting transitive deps too, change the comment here to reflect that
```suggestion
                by any dependent Go builders (direct or indirect) (used to extend
```